lib/resty/core/log.lua

```diff
@@ -0,0 +1,87 @@
+-- Copyright (C) OpenResty Inc.
```

Comment:
Please remove this since the copyright should be granted to the authors of this Lua library.